_posts/2013-02-01-client_libraries.md

```diff
@@ -439,6 +439,19 @@ production? Tell us about it on the [mailing list][mailing_list] or Twitter [@im
     <td></td>
     <td></td>
   </tr>  
+  <tr class="warning">
+    <td><a href="https://github.com/aohan237/asyncnsq">asyncnsq</a></td>
+    <td>Python/Http</td>
```

Comment:
The "/Http" part does not really make sense ... I think you mean "Python/asyncio" but we don't specify "framework" in this column for any other clients, so I'd put just "Python" here.

Reply:
ok..thanks. Python is ok..
in fact, I mean this lib have a asyncio http client.  it add supports for nsq http api